mlir/test/Conversion/MIGraphXToTosa/migraphx-to-rock-conv-nhwc.mlir

```diff
@@ -6,9 +6,9 @@
 // CHECK-SAME: memref<1x3x3x7xf32>
 func.func @convNHWC(%in: !migraphx.shaped<1x4x5x5xf32, 100x1x20x4>, %fil: !migraphx.shaped<7x4x3x3xf32, 36x1x12x4>) -> !migraphx.shaped<1x7x3x3xf32, 63x1x21x7> attributes {kernel, arch = "gfx1100", num_cu = 48 : i64} {
   // CHECK: rock.conv2d
-  // CHECK-SAME: filter_layout = ["k", "y", "x", "c", "g"]
-  // CHECK-SAME: input_layout = ["ni", "hi", "wi", "ci", "gi"]
-  // CHECK-SAME: output_layout = ["no", "ho", "wo", "ko", "go"]
+  // CHECK-SAME: filter_layout = ["g", "k", "y", "x", "c"]
+  // CHECK-SAME: input_layout = ["ni", "hi", "wi", "gi", "ci"]
+  // CHECK-SAME: output_layout = ["no", "ho", "wo", "go", "ko"]
```

Comment:
Why are these changing ?

Reply:
Because we used to just tack the illusory group dimension at the end of the tensor, but now we're actually handling materializing a `g` dimension (if one exists) correctly.